[TASK] Add site config trouble shooting section (#694)
TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted: Documentation/FirstProject/SiteManagement/Index.rst

> .. literalinclude:: _config.yaml
>    :language: yaml
>    :caption: config/sites/example_site/config.yaml

Reviewer: ```suggestion
..  literalinclude:: _config.yaml
    :language: yaml
    :caption: config/sites/example_site/config.yaml
```

..  accordion::
    :name: siteManagementTrouble

    ..  accordion-item:: Set of Site Package not available
        :name: site-trouble-set-unavailable
        :header-level: 2
        :show:

        If the set of your site package is not available make sure, that the
        site package extension is `Installed <https://docs.typo3.org/permalink/t3start:installing-extensions>`_.

        If the site package is installed but the site package set is still not
        available the set or one of its dependencies might be corrupted.

        ..  versionadded:: 13.4.5

        ..  figure:: /FirstProject/SiteManagement/InvalidSiteSet.png
            :alt: Screenshot of the "Site Configuration" module with an invalid site set displayed

            Invalid site sets are listed in the "Site Configuration" module with reasons why they are invalid.

        The site sets are cached. If you change the site set configuration and
        see no difference, `clear the caches <https://docs.typo3.org/permalink/t3start:how-to-clear-cache>`_.

    ..  accordion-item:: This site depends on invalid site sets
        :name: site-trouble-set-invalid
        :header-level: 2

>         If a set or one of its dependencies get corrupted after it was already
>         added to a site configuration, an error box is displayed in the table
>         listing the configurations.

Reviewer: ```suggestion
        If a set or one of its dependencies becomes corrupted after it has
        already been added to a site configuration, an error box appears in
        the table of configurations.
```

        ..  figure:: /FirstProject/SiteManagement/InvalidDependency.png

>             :alt: Screenshot the site configuration module with a site set having an error message "This site depends on invalid site sets"

Reviewer: ```suggestion
            :alt: Screenshot of the Site Configuration module with a site set that displays the error message "This site depends on invalid site sets".
```
